next/components/forms/Groups/DateFromTo.tsx

```diff
@@ -0,0 +1,28 @@
+import cx from 'classnames'
+
+import DatePicker from '../DateTimePicker/DatePicker'
+
+type DatePickerBase = {
+  label: string
+  errorMessage?: string
+}
+
+export const DateFromTo = ({ label, errorMessage }: DatePickerBase) => {
+  return (
+    <div className={cx('flex-col flex items-start p-0 gap-4')}>
+      <div className="flex flex-row items-center p-0 gap-6">
```

Comment:
items shouldn't be centered by default, but left aligned instead. This is true for all the groups